packages/react-components/src/components/Switch/Switch.tsx

```diff
@@ -73,7 +112,7 @@ export const Switch: React.FC<SwitchProps> = ({
         checked={checked}
         name={name}
         ref={innerRef}
-        disabled={disabled}
+        disabled={shouldBehaveAsDisabled}
```

Comment:
What is the expected `pointer` in case of loading?

Reply:
I'm waiting for @martagranisz for that info